src/stores/SessionStore.js

```diff
@@ -66,6 +66,11 @@ class SessionStore extends Store {
                     cachedPassword: null,
                 });
                 break;
+            case 'on_logged_out':
+                this._state = {
+                    cachedPassword: null,
```

Comment:
this duplication feels pretty ming and fragile, surely we should use the same pattern as RoomViewStore and have an INITIAL_STATE variable which gets copied over the top of this._state.